News and team page

EgorK10 opened the pull request.

TasyaKh commented:
@EgorK10  пул реквест не туда прилетел, надо было в RatingTeamEvents2, если ты в VSCode там пример отправляла с пул реквестом

TasyaKh committed:
feat: add packages

AnnaVyvert committed:
chore(release-tags): devops-tools

AnnaVyvert committed:
Merge branch 'main' of github.com:TasyaKh/Raiting-Teams-Events2

AnnaVyvert committed:
ci: 🎡 update linter rules, linters, husky

AnnaVyvert committed:
feat(configuration): 🌟 setup cz config

AnnaVyvert committed:
fix(configuration): 🐛 add spaces for readability

TasyaKh committed:
fix: исправлено дублирование разрешений и ошибки при null permissions in user

TasyaKh committed:
fix: при сохранении изменений чтобы списки сразу обновились

TasyaKh committed:
fix: align tags center

TasyaKh committed:
feat: new backup

TasyaKh committed:
Merge branch 'main' into feature/change-user-permissions

TasyaKh committed:
chore(release-tags): be

TasyaKh committed:
chore(release-tags): fe

TasyaKh committed:
docs: new logic/physical model

TasyaKh committed:
Merge branch 'main' into fix/143-device-adaptation

rikisukamaru committed:
Сделал редактирование

TasyaKh committed:
fix: pagination make page less for phones

TasyaKh committed:
fix: del unused things (package sharp and code)

TasyaKh committed:
fix: add public and media

TasyaKh committed:
Merge branch 'main' into feature/admin-panel-resp-derection

EgorK10 committed:
Add buttons

Сделал кнопки и фильтр
И почти сделал прекрасный календарь (нет)

AnnaVyvert committed:
fix: make merge

TasyaKh committed:
chore(release-tags): fe

TasyaKh committed:
chore(release-tags): be

TasyaKh committed:
feat: add comment field in db

TasyaKh committed:
fix: merge

TasyaKh committed:
fix: merge

TasyaKh committed:
fix: resize img without cropping

TasyaKh committed:
fix: resize img without cropping

TasyaKh committed:
chore(release-tags): be

AnnaVyvert committed:
docs(readme): 📚 update ci/cd info

добавил информацию по git cz, линтёры, проверку хуками

TasyaKh committed:
fix: we already have 2 datepickers v-calendar and @vuepic/vue-datepicker

TasyaKh committed:
fix: merge main

TasyaKh committed:
fix: change search and set check team? to each values

TasyaKh committed:
fix: при увелич карточки учатсника убрать фото

TasyaKh committed:
fix(team-news): поправить календарь и списки

TasyaKh committed:
fix(teams-news): calendar when click end date hide calendar

TasyaKh committed:
Merge branch 'main' into feature/news-page

TasyaKh committed:
fix: nav second border make smaller

TasyaKh committed:
fix(team-requests): alert change placement

TasyaKh committed:
test: 👷 check git cz from web storm

TasyaKh committed:
test(FE/assets): 👷 without spaces test git cz

TasyaKh committed:
test: 👷 git cz found conflicts husky and git cz

если использовать test(FE / assets): ....... то хаски ругается на
пробелы

TasyaKh committed:
fix: 🐛 change btn color ankete

TasyaKh committed:
fix: 🐛 change btn color ankete

TasyaKh committed:
chore(release-tags): 🔧 fe up

rikisukamaru committed:
пофиксил баги

AnnaVyvert committed:
fix(configuration): git cz, remove spaces for husky linter

TasyaKh committed:
feat: 🌟 add comment in requisition and make it update for leader

TasyaKh committed:
feat: 🌟 add comment field, and fix query in team service

TasyaKh committed:
fix: 🐛 comment btn only one field

TasyaKh committed:
fix: 🐛 comment field in tem requests not dublicate

TasyaKh committed:
Merge branch 'main' into feature/requisition-add-comment

TasyaKh committed:
refactor: merge main

TasyaKh committed:
fix(FE/components): 🐛 toggle forbidden use Function word in props

TasyaKh committed:
fix(FE/views): 🐛 change edit team icon and size cards on phones

TasyaKh committed:
chore(release-tags): 🔧 be up

TasyaKh committed:
chore(release-tags): 🔧 fe up

TasyaKh committed:
feat(BE/teams): 🌟 upgrade query filter /:team_id/requisition

added filters:  params.statuses[], date_update_order ("ASC" | "DESC"),
fullname user, user_id

TasyaKh committed:
feat(FE/views): 🌟 filters on team requests now is workig

TasyaKh committed:
feat(FE/views): 🌟 filters on team requests now is workig

TasyaKh committed:
fix(FE/views): 🐛 505 error in modal questionaire set requis.team_id=idTeam

TasyaKh committed:
fix(FE/views): 🐛 on reload page requests is empty

TasyaKh committed:
chore(release-tags): 🔧 be up

TasyaKh committed:
chore(release-tags): 🔧 fe up

TasyaKh committed:
feat(BE/events): 🌟 added filter order date and by journal

TasyaKh committed:
feat(FE/views): 🌟 filters on team news added

TasyaKh committed:
fix: 🐛 add loading elem and alert if data is empty

rikisukamaru committed:
Создал таблицу посещаемости+календарь на неделю

TasyaKh committed:
feat(FE/views): 🌟 add pagination in team news

TasyaKh committed:
fix(FE/components): 🐛 pagination end start arrows add disable state

TasyaKh committed:
fix(FE/views): 🐛 team news update limit = 5

TasyaKh committed:
chore(release-tags): 🔧 fe up

TasyaKh committed:
chore(release-tags): be up

AnnaVyvert committed:
feat(docker): 🌟 init

AnnaVyvert committed:
feat(docker): 🌟 set env vars for docker

AnnaVyvert committed:
docs(readme): 📚 docker info

AnnaVyvert committed:
feat(docker): 🌟 script for db restoring

AnnaVyvert committed:
feat(readme): 🌟 add info about package manager in the future, docker

TasyaKh committed:
feat: 🌟 merge and resolve bugs, make filters

TasyaKh committed:
fix: reupdate all

TasyaKh committed:
fix(FE/views): 🐛 pagination make

TasyaKh committed:
feat: 🌟 add in usr funct dto limit and offset

TasyaKh committed:
fix(FE/views): 🐛 add dropdown for delete user from team

TasyaKh committed:
feat: merge teams cards

TasyaKh committed:
fix: madd avatar img

TasyaKh committed:
fix: 🐛 beautify form elems

TasyaKh committed:
fix: add db in gitignore

TasyaKh committed:
fix: rm db from gitignore

TasyaKh committed:
chore(release-tags): 🔧 be up

TasyaKh committed:
chore(release-tags): 🔧 fe up

AnnaVyvert committed:
feat(docker): 🌟 add dockerignore

TasyaKh committed:
feat(BE/teams): 🌟 add schedule, visits and auditories in team db

TasyaKh committed:
feat(BE/teams): 🌟 add schedule, visits and auditories in team db

TasyaKh committed:
feat: add auditories in team db

TasyaKh committed:
feat: 🌟 add comment field in user team requests

TasyaKh committed:
fix: teams without description

AnnaVyvert committed:
chore(release-tags): 🔧 init docker

TasyaKh committed:
fix: events del description

TasyaKh committed:
chore(release-tags): 🔧 fe up

TasyaKh committed:
chore(release-tags): 🔧 be up

TasyaKh committed:
Merge branch 'main' into feature/docker-iteration-1

# Conflicts:
#	release-tags.json

AnnaVyvert committed:
refactor(docker): 💡 update configuration with additional env vars

AnnaVyvert committed:
refactor(docker): 💡 db improvements

AnnaVyvert committed:
refactor(docker): 💡 be improvements

AnnaVyvert committed:
refactor(docker): 💡 fe improvements

AnnaVyvert committed:
feat(docker): 🌟 set ignores

AnnaVyvert committed:
chore(release-tags): 🔧 docker

TasyaKh committed:
fix: add backup_last.sql

AnnaVyvert committed:
fix(docker): 🐛 update db dump

AnnaVyvert committed:
chore(docker): 🔧 remove root .env

AnnaVyvert committed:
docs(readme): 📚 docker info

AnnaVyvert committed:
Merge pull request #34 from TrainForest/fix/docker-improvements

Fix/docker improvements

снизить размер образов
рефакторить конфиг контейнеризации БД
прокинуть переменные для создания БД
починить префиксы
удалить .env из трекинга

образ pg 1.3gb -> 0.2gb
образ be 1.72gb -> 0.5gb
образ fe 2gb -> 1.6gb

TasyaKh committed:
feat: можно подать заявку на вступление в колелктив

TasyaKh committed:
feat: можно подать заявку на вступление в колелктив

TasyaKh committed:
feat: added request in team delete

TasyaKh committed:
fix: fix syntax

AkitaSp committed:
chore(release-tags): 🔧 be up

AkitaSp committed:
chore(release-tags): 🔧 fe up

AkitaSp committed:
Merge branch 'main' into feature/db-add-schedule

# Conflicts:
#	Схема БД.drawio

PashaMashina committed:
feat: add backup and new fields in team visits comment, status_visit

AkitaSp committed:
fix: linter

AkitaSp committed:
feat(BE/teams): 🌟 feat: added get query for team visits, created migration

AkitaSp committed:
feat: 🌟 new db dump, and schedule folder

AkitaSp committed:
feat: new dump

AkitaSp committed:
chore(release-tags): 🔧 be up

bot-irnitu committed:
chore: vps db bak

AkitaSp committed:
fix: env.example add BITRIX_CLIENT_ID

AkitaSp committed:
feat: 🌟 added time for cabinets entity

AkitaSp committed:
feat: cabinets entity

AkitaSp committed:
fix: del warnings imports

AkitaSp committed:
feat: 🌟 added post visits

AnnaVyvert committed:
fix(DB): 🐛 last production DB (from pavelevtuh)

AnnaVyvert committed:
feat(DB): 🌟 tools for db dumping

AnnaVyvert committed:
Merge branch 'main' into feature/db-tools

AnnaVyvert committed:
fix(BE): 🐛 unused vars linter

AnnaVyvert committed:
chore(env): 🔧 update client env

AnnaVyvert committed:
feat(configuration): 🌟 add just FE entity to CL.config

AnnaVyvert committed:
chore(release-tags): 🔧 update BE, devops-tools

AkitaSp committed:
chore: new dump

AkitaSp committed:
Merge remote-tracking branch 'origin/main' into feature/db-add-schedule

AkitaSp committed:
chore(release-tags): 🔧 be up

EgorGordinskiy committed:
Feature/cabinets

AkitaSp committed:
fix: delete move id in params url and create tests

AkitaSp committed:
fix: eslint

AkitaSp committed:
chore(release-tags): 🔧 be up

AkitaSp committed:
Merge branch 'main' into feature/db-tools

# Conflicts:
#	client/.env.example
#	db/2024-02-24-09.45.44_vps.sql
#	release-tags.json
#	server/src/forms/forms.service.ts
#	server/src/schedule/schedule.controller.ts
#	server/src/teams/teams.service.ts

AkitaSp committed:
chore: dump db

AkitaSp committed:
feat: add in cabinets (direction and tags)

AkitaSp committed:
feat: add in cabinets (direction and tags)

AkitaSp committed:
feat: add cabinets seach model and store

AkitaSp committed:
feat: add cabinets seach model and store(added auditories)

AkitaSp committed:
feat: add cabinets find by ids

AkitaSp committed:
fix: eslint

AkitaSp committed:
fix: value?.id is null

AkitaSp committed:
chore: update dump

AkitaSp committed:
chore(release-tags): 🔧 fe up

AkitaSp committed:
chore(release-tags): 🔧 be up

AkitaSp committed:
Merge branch 'main' into fix/143-device-adaptation

# Conflicts:
#	client/index.html
#	client/src/components/PaginationElem.vue

AkitaSp committed:
fix: participation overflow-scroll table on grid

AkitaSp committed:
chore(release-tags): 🔧 fe up

AkitaSp committed:
feat: can assign multiple leaders in team

AkitaSp committed:
fix: in direction view all leaders

AkitaSp committed:
chore(release-tags): 🔧 fe up

AkitaSp committed:
chore(release-tags): 🔧 be up

AkitaSp committed:
fix: send requisition if form is empty

AkitaSp committed:
fix: bug on front when archive team

AkitaSp committed:
fix: bug on create team without leaders

AkitaSp committed:
fix: another form bug when creating form we need add team

AkitaSp committed:
fix: adaptive form

AnnaVyvert committed:
temp(FE/views): 🚧 schedule add class page

AkitaSp committed:
fix: cards teams make edit button absolute position

AkitaSp committed:
fix: add alt to images

AkitaSp committed:
chore(release-tags): 🔧 fe up

AkitaSp committed:
chore(release-tags): 🔧 be up

AkitaSp committed:
feat: we can add links, phone nd photos in team

AnnaVyvert committed:
feat(FE/views): 🌟 select and switch

AnnaVyvert committed:
feat(FE/common): 🌟 group elements component for add class form

AkitaSp committed:
feat: we can add/delete team avatars and photos

AkitaSp committed:
feat: leader of team can edit team

AkitaSp committed:
fix: 🐛 delete avarats of team filter

AkitaSp committed:
fix: 🐛 when go between tabs team data became empty

AkitaSp committed:
feat: add photo is working

AkitaSp committed:
fix: getTeamPhoto .where('team_photos.id = :id', { id: id })

AkitaSp committed:
fix: add title to each field edit team

AnnaVyvert committed:
feat(configuration): 🌟 add default run script

AnnaVyvert committed:
feat(FE/common): 🌟 month enums

AnnaVyvert committed:
feat(FE/common): 🌟 add short week days

AnnaVyvert committed:
feat(FE/views): 🌟 book class component with children components

AkitaSp committed:
chore(release-tags): 🔧 be up

AkitaSp committed:
chore(release-tags): 🔧 fe up

AkitaSp committed:
chore: new backup

AkitaSp committed:
feat: can get schedule by team id and get cabinets

AkitaSp committed:
feat: add search cabinet by tag

AkitaSp committed:
fix: test queries and createCabinetTime check ids

AkitaSp committed:
fix: rename   @Post('cabinets-time')

AkitaSp committed:
feat: update query

hawkavis committed:
feat(FE/views): 🌟 styles for class add component

hawkavis committed:
fix(FE/views): change star

AkitaSp committed:
feat: auto create schedule for team on team create

AkitaSp committed:
chore: new backup

AkitaSp committed:
chore: new backup

AkitaSp committed:
chore(release-tags): 🔧 be up

hawkavis committed:
fix(FE/views): styles for schedule and uppercase for header

hawkavis committed:
fix(FE/views): remove usless class

AkitaSp committed:
fix: justify schedule by center

AnnaVyvert committed:
feat(FE/views): 🌟 class list card component

AnnaVyvert committed:
fix(FE/views): 🐛 num of class list card component

AkitaSp committed:
feat: add buttons setting for cells in schedule

AkitaSp committed:
fix: create new time if there is none

AkitaSp committed:
chore(release-tags): 🔧 beup

AkitaSp committed:
chore(release-tags): 🔧 fe up

EgorGordinskiy committed:
Feature/create-team-schedule

hawkavis committed:
fix(FE/views): fonts and margins layout

EgorK10 committed:
Create AuditoriumField

hawkavis committed:
fix(FE/views): minor layout change

AkitaSp committed:
test: http test schedule post

AkitaSp committed:
fix: change path for file

AkitaSp committed:
fix: when click on search btn in schedule open auditories

AnnaVyvert committed:
feat(FE/views): 🌟 style class list cards

AkitaSp committed:
fix: when new participant then old rewrite

AkitaSp committed:
fix: when new participant then old rewrite

AkitaSp committed:
chore(release-tags): 🔧 be up

AkitaSp committed:
feat: can search auditory by name

AkitaSp committed:
Merge branch 'feature/auditories-window' into feature/PREPARE-PROJECT

hawkavis committed:
fix(FE/views): minor layout change

AkitaSp committed:
feat: we can see free cabinets, search them

AkitaSp committed:
feat: get cabinet schedule by cabinet id

AkitaSp committed:
fix:         :handle-on-auditory-click="onAuditoryClick"/>

AkitaSp committed:
fix:   timeDayWeek rm duplicates

AnnaVyvert committed:
fix(FE/views): 🐛 class booking, schedule create page styles

EgorGordinskiy committed:
fix

AnnaVyvert committed:
feat(FE/views): 🌟 modal, chip padding token, genitive case months, fix v-deep

AnnaVyvert committed:
feat(FE/views): 🌟 modal, chip padding token, genitive case months, fix v-deep

hawkavis committed:
fix(FE/views): 🐛 class booking page styles

AkitaSp committed:
chore: update db scheme

AkitaSp committed:
chore: update db

AkitaSp committed:
chore(release-tags): be up

AkitaSp committed:
chore(release-tags): fe up

AkitaSp committed:
fix: delete unusing file

AkitaSp committed:
Merge branch 'main' into feature/create-team-schedule

AkitaSp committed:
chore(release-tags): be up

hawkavis committed:
fix(FE/views): schedule create page styles

AnnaVyvert committed:
fix(FE/views): 🐛 width of column elements, width of page and its position

AkitaSp committed:
Merge remote-tracking branch 'origin/main' into feature/schedule-add-class-page

AkitaSp committed:
feat: get cabinets from database

AkitaSp committed:
chore(release-tags): fe up

AkitaSp committed:
Merge remote-tracking branch 'origin/feature/schedule-add-class-page' into feature/schedule-add-class-page

# Conflicts:
#	client/src/views/schedule/create-page/ClassListCard.vue
#	client/src/views/schedule/create-page/ScheduleCreate.vue
#	client/src/views/teams/schedule/create-page/group-option.enum.ts
#	client/src/views/teams/schedule/create-page/type-option.enum.ts

AkitaSp committed:
Merge branch 'feature/schedule-add-class-page'

AkitaSp committed:
chore(release-tags): fe up
